### PR TITLE
#4 Hero and columns component

### LINKED ISSUE #4: Hero

### PR DESCRIPTION
**Pull request by SantiagoHomps-NC**
This PR is done with not only the Hero Component but also Columns taking the **about-mac.doc** as an example for content. 

Fix #4

Test URLs:
- Before: https://main--vg-macktrucks-com--hlxsites.hlx.page/about-mack/
- After: https://4-hero--vg-macktrucks-com--hlxsites.hlx.page/about-mack/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -162,9 +188,9 @@ main input:hover {
   border: 1px solid var(--text-color);
 }
 
-main .section {
```
Comment (jonatan-lledo-netcentric) [inline styles/styles.css:165]: remember to delete commented code, please

File: blocks/hero/hero.css [hunk 2]
```diff
@@ -1,40 +1,96 @@
...
     right: 0;
-    object-fit: cover;
     box-sizing: border-box;
 }
 
-main .hero img {
+.hero img {
     object-fit: cover;
-    width: 100%;
+    width: 105%;
```
[the remark this answers is not in the document]
Comment (SantiagoHomps-NC) [reply blocks/hero/hero.css:47]: its how it was on _macktrucks.com_ site 🤷

File: blocks/hero/hero.css [hunk 3]
```diff
@@ -1,40 +1,96 @@
...
+
+    .hero h1 {
+        font-size: 40px;
+        margin-bottom: 11px;
+        line-height: 38px;
+    }
+    
+    .hero p {
+        font-size: 18px;
+        margin-bottom: 18px;
```
[the remark this answers is not in the document]
Comment (SantiagoHomps-NC) [reply blocks/hero/hero.css:93]: done

File: blocks/columns/columns.css [hunk 4]
```diff
@@ -7,19 +8,100 @@
...
+    line-height: normal;
+    margin-bottom: 6px;
+}
+
+.item-post a:hover {
+    color: var(--text-color-hover);
+    text-decoration: none;
+}
+
+@media (max-width: 767px) {
```
[the remark this answers is not in the document]
Comment (SantiagoHomps-NC) [reply blocks/columns/columns.css:89]: you are right... guess it's what I was used to. I can update it later

**Review by jonatan-lledo-netcentric — APPROVED** [review]

**Pull request closed without merging**